Library/Formula/bacula-fd.rb

```diff
@@ -1,8 +1,8 @@
 class BaculaFd < Formula
   desc "Network backup solution"
   homepage "http://www.bacula.org/"
-  url "https://downloads.sourceforge.net/project/bacula/bacula/7.0.5/bacula-7.0.5.tar.gz"
-  sha256 "1457849eb33011b43371801b62ffa13d29bebe51be8d5a36da563b87bb094a49"
+  url "https://downloads.sourceforge.net/project/bacula/bacula/7.4.0/bacula-7.4.0.tar.gz"
+  sha256 "fe850b783523edb19fb4dbfa8c44752d20955121b71a52b0740a9e765bfd73cb"
 
   bottle do
     revision 1
```

Comment:
Should probably remove the bottle's `revision 1` since we're bumping the formula version. Or does the bottling process do that automatically?